docs/scu/dev/features/upgrade-verification.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Active detection of incompatibilies
+description: How Spryker Code Upgrader actively detects and warns you when your code becomes incompatible with the code in upgraded modules
+template: concept-topic-template
+---
+
+It is very important for Spryker Code Upgrader to ensure stability of the upgrades. On the one hand you have your automated tests that you rely on, those tests are run in your CI system and you use them before you merge the pull request created by Spryker Code Upgrader. On the other hand and in addition we run a number of code checks to offer an early warning system.
+
+# Validations and warnings
+
+The upgrader offers validations ensure the accuracy and safety of the upgrade process.
+The warnings provide guidance on potential risks or necessary actions for a successful upgrade.
+
+Warning types:
```

Comment:
1. Are these warnings provided before or after merging a PR?
2. Where are they provided?

Reply:
1. Are provided before merging the PR. 
2. Warnings are placed in PR description text area